### PR TITLE
Fix issue #1104 - No spam on da click peep

### LINKED ISSUE #1104: On slow connection ~250 kb/s (2G) Stripe checkout can be opened multiple times
Stripe checkout appears to have a slight loading delay and on slow connections, you can mash the Credit/debit card button to open multiple modal dialogs.

I can see this confusing impatient donors who aren't expecting multiple dialogs.

We should disable the click action until the stripe checkout model closes (i.e. flow is cancelled)

### PR DESCRIPTION
**Pull request by alicoding**
Closes #1104

### CODE REVIEW

**Comment by alicoding** [comment]
@ScottDowne R?

File: mixins/form.jsx [hunk 1]
```diff
@@ -292,6 +296,11 @@ module.exports = {
       zipCode: true,
       billingAddress: billingAddress,
       locale: locale,
+      closed: function() {
+        this.setState({
```
Comment (ScottDowne) [inline mixins/form.jsx]: `this` isn't the expected this while inside the callback.

**Comment by ScottDowne** [comment]
Small issue.

**Comment by alicoding** [comment]
R+ by scott in person

**Pull request merged**